docs/rule/no-at-ember-render-modifiers.md

```diff
@@ -0,0 +1,50 @@
+# no-at-ember-render-modifiers
+
+`@ember/render-modifiers` were meant as a transitional tool from the pre-Octane era, and not long-term usage.
+
+More information about this can be found on the [`@ember/render-modifiers` README](https://github.com/emberjs/ember-render-modifiers#when-to-use-these-modifiers-and-when-not-to-use-them)
```

Comment:
```suggestion
More information about this can be found on the [`@ember/render-modifiers` README](https://github.com/emberjs/ember-render-modifiers#when-to-use-these-modifiers-and-when-not-to-use-them).
```